### PR TITLE
fix width in publications

### PR DESCRIPTION
**Pull request by kbaley**
Fixes issues #45 and #46 

See screenshots

## Small view

screen shot 2015-07-04 at 9 23 37 pm
screen shot 2015-07-04 at 9 23 32 pm

## Medium size view

screen shot 2015-07-04 at 9 20 11 pm

## Largest view

screen shot 2015-07-04 at 9 19 53 pm
screen shot 2015-07-04 at 9 19 32 pm

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -1,204 +0,0 @@
-/**
```
Comment (kbaley) [inline _sass/_base.scss]: These deleted files were left over from the previous theme.

File: _sass/_publications.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@
   
   .pill {
     display: inline-block;
-    width: 100px;
+    width: 25%;
```
Comment (dpaquette) [inline _sass/_publications.scss]: How does this look on a small screen /  phone?

Comment (kbaley) [inline _sass/_publications.scss]: I think one of the screenshots in the PR description shows it on the
smallest size

Comment (dpaquette) [inline _sass/_publications.scss]: ah, sorry. I thought that first one was what it used to look like on a big screen

**Comment by dpaquette** [comment]
Good improvements! :shipit:

**Pull request merged**